packages/frontend-2/components/viewer/explorer/Filters.vue

```diff
@@ -248,4 +234,27 @@ const refreshColorsIfSetOrActiveFilterIsNumeric = () => {
   // removePropertyFilter()
   applyPropertyFilter()
 }
+
+const isRevitProperty = (key: string): boolean => {
+  return revitPropertyRegex.test(key)
+}
+
+const getPropertyName = (key: string): string => {
```

Comment:
Seems like everywhere where we use this fn we have access to the entire filter object. If we pass that in, instead of just the key, we don't have to do the `props.filters.find()` bit

Reply:
Passing the entire filter object introduced complexity with a fallback mechanism and error handling in title. Let me know if this is enough to stop us merging and I can take another look on Monday but I'm running out of time today